### PR TITLE
[JENKINS-48546] - Restore the binary compatibility with 2.9

### PR DESCRIPTION
**Pull request by oleg-nenashev**
I missed a binary compatibility issue during the code reviews. We should employ more robots to do that, right? :(

https://issues.jenkins-ci.org/browse/JENKINS-48546

@reviewbybees @jglick

### CODE REVIEW

**Comment by jglick** [comment]
I guess this is another regression from #198? CC @zionyx

**Comment by oleg-nenashev** [comment]
It is

File: src/main/java/hudson/scm/SubversionSCM.java [hunk 1]
```diff
@@ -979,7 +979,7 @@ public boolean requiresWorkspaceForPolling() {
     private static class CheckOutTask extends UpdateTask implements FileCallable<List<External>> {
         private final UpdateTask task;
 
-        public CheckOutTask(Run<?, ?> build, SubversionSCM parent, ModuleLocation location, Date timestamp,
+        CheckOutTask(Run<?, ?> build, SubversionSCM parent, ModuleLocation location, Date timestamp,
```
Comment (jglick) [inline src/main/java/hudson/scm/SubversionSCM.java]: Unrelated I guess.

Comment (oleg-nenashev) [reply src/main/java/hudson/scm/SubversionSCM.java]: just wanted to make it explicit, because the method signature has also changed

**Comment by oleg-nenashev** [comment]
@reviewbybees done
🚢 🇮🇹

**Pull request merged**